src/table/table.scss

```diff
@@ -35,54 +35,62 @@
     border-right: solid 1px transparent;
   }
 
-  .iui-cell:not(.iui-slot) {
-    @include iui-focus;
-
-    &.iui-actionable {
-      cursor: pointer;
+  .iui-cell {
+    &-label {
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
```

Comment:
I don't think this is a proper fix. 
![image](https://user-images.githubusercontent.com/9084735/166313502-7b520708-4756-4fee-9e98-a2d7adc1375f.png)

What was wrong with `flex-wrap`? I know it wouldn't automagically handle the sort icon, but that was part of the task.

Reply:
I first tried applying `white-space: nowrap; flex-wrap: wrap;` but wasn't very happy with how it was looking.
![Screen Shot 2022-05-02 at 12 51 59 PM](https://user-images.githubusercontent.com/849817/166314617-a68d2b6a-0086-433a-ae50-d35fed12371b.png)

Reply:
I agree with Mayank, truncated headers doesn't give any info to the user.